### ISSUE TITLE
MultipleCompilationErrorsException: startup failed:

### ISSUE THREAD

**Issue by rtyler** [CodeValet/codevalet#82]
https://sentry.io/codevalet/codevalet/issues/396988829/

```
MultipleCompilationErrorsException: startup failed:
/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 59: unable to resolve class AzureVMCloud 
 @ line 59, column 13.
       cloud = new AzureVMCloud(cloudName,         /* Cloud Name */
               ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 83: unable to resolve class AzureVMCloudRetensionStrategy 
 @ line 83, column 21.
       retentionTime = new AzureVMCloudRetensionStrategy(10)
                       ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 84: unable to resolve class AzureVMAgentTemplate.ImageReferenceTypeClass 
 @ line 84, column 22.
       imageReference = new AzureVMAgentTemplate.ImageReferenceTypeClass(vhd, vhd, vhd, vhd, vhd)
                        ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 85: unable to resolve class AzureVMAgentTemplate 
 @ line 85, column 18.
       vmTemplate = new AzureVMAgentTemplate('windows-2016-docker',
                    ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 129: unable to resolve class AzureVMCloudRetensionStrategy 
 @ line 129, column 21.
       retentionTime = new AzureVMCloudRetensionStrategy(10)
                       ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 130: unable to resolve class AzureVMAgentTemplate.ImageReferenceTypeClass 
 @ line 130, column 22.
       imageReference = new AzureVMAgentTemplate.ImageReferenceTypeClass(vhd, vhd, vhd, vhd, vhd)
                        ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 131: unable to resolve class AzureVMAgentTemplate 
 @ line 131, column 18.
       vmTemplate = new AzureVMAgentTemplate('docker-ubuntu',
                    ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 175: unable to resolve class AzureVMCloudRetensionStrategy 
 @ line 175, column 21.
       retentionTime = new AzureVMCloudRetensionStrategy(10)
                       ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 176: unable to resolve class AzureVMAgentTemplate.ImageReferenceTypeClass 
 @ line 176, column 22.
       imageReference = new AzureVMAgentTemplate.ImageReferenceTypeClass(vhd, vhd, vhd, vhd, vhd)
                        ^

/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy: 177: unable to resolve class AzureVMAgentTemplate 
 @ line 177, column 18.
       vmTemplate = new AzureVMAgentTemplate('freebsd-11',
                    ^

10 errors

    at org.codehaus.groovy.control.ErrorCollector.failIfErrors(ErrorCollector.java:310)
    at org.codehaus.groovy.control.CompilationUnit.applyToSourceUnits(CompilationUnit.java:958)
    at org.codehaus.groovy.control.CompilationUnit.doPhaseOperation(CompilationUnit.java:605)
    at org.codehaus.groovy.control.CompilationUnit.compile(CompilationUnit.java:554)
    at groovy.lang.GroovyClassLoader.doParseClass(GroovyClassLoader.java:298)
...
(21 additional frame(s) were not displayed)

Failed to run script file:/var/jenkins_home/init.groovy.d/setup-azure-cloud.groovy
```

**Comment by rtyler** [comment]
Looks like more fun internal API changes in the Azure VM Agent plugin

**Comment by rtyler** [comment]
Fixed with codevalet/master@49cb58e2bb685ae4b0d7148392c68ed219bc601c and a rebuild
